modules/ppcp-button/resources/js/modules/ActionHandler/CheckoutActionHandler.js

```diff
@@ -48,7 +48,11 @@ class CheckoutActionHandler {
                                 .querySelector('ul')
                         );
                     } else {
-                        errorHandler.message(data.data.message, true);
+                        if (data.data.details.length > 0) {
+                            errorHandler.message(data.data.details[0].issue + ' ' + data.data.details[0].description, true);
```

Comment:
should we displays only the first one? Maybe better e.g. 

    data.data.details.map(d => `${d.issue} ${d.description}`).join('<br/>')

Reply:
@AlexP11223 Done, thanks!